fmt.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: fmt
-  version: 10.2.1
```

Comment:
Hello y’all! We’ve run into an issue where the major version change in fmt, 10.2.1 to 11.0.0 has broken the ability to compile fmt with mamba/libmamba. I am reaching out for eyes with more cpp experience to give insight into if this is something we have the capabilities and bandwidth to address as a patch ourselves or if this is an issue that needs to be addressed upstream.
Here is a link to the [actual error itself](https://github.com/wolfi-dev/os/actions/runs/9947096519/job/27479081396?pr=23233), the fmt [version change log](https://github.com/fmtlib/fmt/releases/tag/11.0.0) and the [commit history](https://github.com/wolfi-dev/os/pull/23233) of this commit
From what I have looked into these are my concerns.
fmt::join has had its range and iterator overloads moved to fmt/ranges.h. which in an of itself isn’t an issue and could just be a simple import that doesn’t exist currently in the file.
11.0.0 also enabled ADL for begin and end. If I am not mistaken, changing the scope of function lookup might change how iterators are resolved in fmt::join. Is there anyone who has more detailed knowledge of cpp that can verify either way?
Looks like there are some changes to the way the api handles the table.add_row function as the version that was compatible with 10.2.1 no longer satisfies the argument requirements. I didn't feel comfortable blindly attempting the [suggested fix in the build failed](https://github.com/wolfi-dev/os/pull/23233#:~:text=cpp%60.%0A2.%20Ensure%20%60CLI/StringTools.hpp%60%20is%20included.%0A3.-,Convert%20the%20initializer%20list%20to%20a,-%60std%3A%3Avector%3Cmamba%3A%3Aprinters%3A%3AFormattedString%3E%60%20before%20passing%20to%20%60table) commit without investigating and understanding the changes that have been made.